Cargo.toml

```diff
@@ -114,25 +115,25 @@ rustls-pemfile = "2.0"
 rustls-pki-types = "1.11"
 script_layout_interface = { path = "components/shared/script_layout" }
 script_traits = { path = "components/shared/script" }
-selectors = { git = "https://github.com/servo/stylo", branch = "2025-01-02" }
+selectors = { git = "https://github.com/servo/stylo", rev = "refs/pull/110/head" }
```

Comment:
What's the process here? Should I wait for the next Stylo branch?

Reply:
Just leave `Cargo.toml` unchanged (so `branch = "2025-02-03"`) but with `Cargo.lock` pointing to the tip (8af5d515a03bfca5d19fcaa541783327673a951e)

See https://github.com/servo/servo/pull/35352/files for reference

Reply:
Done, thanks!